src/fileio/resmgr/res_image.cpp

```diff
@@ -8,12 +8,20 @@
 
 #include "res_image.hpp"
 
-class ImageLoader : public ResMgr::cLoader<sf::Texture> {
+#include "gfx/texture.hpp"
+
+class TextureLoader : public ResMgr::cLoader<Texture> {
```

Comment:
I don't want to accept this patch.

I'm not sure what it does, and a different patch that does a similar thing may be acceptable as long as I can understand its purpose.

However, this patch renames a whole bunch of things that didn't really need to be renamed at all. You could have easily rewritten the internals of this loader without renaming it, its instance, or the `using` typedef for the resource pool's return type.

If you want to resubmit this later after renaming all those things (and anything else you renamed) back to their original names, and you can explain what it does and why, then I'll consider it. But I won't accept it its current form.

Also, if you do end up doing that, please submit it in a pull request all on its own.

Reply:
For me (and SFML), we are mainly reading textures here. And using ImageLoader with ResMgr::graphics doesn't seem consistent.

But I'll try to rewrite this code to keep the old name (when I find time to do it).

Reply:
For reference, this is the specific commit in case you're having trouble finding it: 848325b6eeafa33e20b074cafb352edb944a549b